src/main.cpp

```diff
@@ -4591,17 +4660,28 @@ bool ReceivedBlockTransactions(
             CBlockIndex *pindex = queue.front();
             queue.pop_front();
             pindex->nChainTx = (pindex->pprev ? pindex->pprev->nChainTx : 0) + pindex->nTx;
+
             if (pindex->pprev) {
+                // Transparent value and chain total supply are added to the
+                // block index only in `ConnectBlock`, because that's the only
+                // place that we have a valid coins view with which to compute
+                // the transparent input value and fees.
+
+                // calculate the block's effect on the chain's net Sprout value
```

Comment:
```suggestion
                // Calculate the block's effect on the Sprout chain value pool balance.
```
This is the term used in ZIP 209. Similarly for the other chain value pools below.